dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/beans/factory/annotation/DubboConfigBindingBeanPostProcessor.java

```diff
@@ -126,8 +128,7 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
         this.applicationContext = applicationContext;
     }
 
-    @Override
-    public void afterPropertiesSet() throws Exception {
+    public void init() {
```

Comment:
If we allow to create sub class of this **DubboConfigBindingBeanPostProcessor ** and some **init method** get override then it might cause issue. So better to make init final or provide.

Reply:
How about make it pirate?

Reply:
That should work. Thanks for your consideration.